frontend/src/components/ResourceInfo.test.tsx

```diff
@@ -17,7 +17,7 @@
 import * as React from 'react';
 import { ResourceInfo, ResourceType } from './ResourceInfo';
 import { getByTestId, render, screen } from '@testing-library/react';
-import { Artifact, Value } from '@kubeflow/frontend';
+import { Artifact, Value } from '../mlmd/library';
```

Comment:
nit: it'd be easier to migrate if you use absolute path to import: `import 'src/mlmd/library'`

Reply:
Make sense, I will update all the import paths.